Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
```

Comment:
`error` can be omitted (and usually is). In that case we should use sys.exc_info() to get the last exception and check that is in not `NotImplementedError`

Reply:
done

Reply:
excellent